modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -56,163 +52,166 @@ For example:
+** An external load balancer must be configured to load balance incoming requests to the controller nodes, with a health check configured for TCP port 443.
+** The CPU and memory resources for at least one additional worker node must be available in the cluster.
 +
 This enables safe removal of a worker node from the cluster to apply upgrades without impacting application availability.
-*** Applications serving inbound requests must be scaled to a minimum of two replicas.
-+
-[WARNING] Runtime Fabrics running on cluster version 1.0.x encounter application downtime if configured to use an HTTP proxy. You must apply the HTTP proxy configuration on each node after the upgrade. Locate the Runtime Fabric version by running `rtfctl version` on a node and referencing the appliance version.
+** Applications serving inbound requests must be scaled to a minimum of two replicas.
 
-. Update the Mule deployments running on Runtime Fabric to the latest patch version.
-.. Using your web browser, navigate to the *Applications* tab in Runtime Manager.
-.. Select an application and select *Manage Application*. 
-.. The value in the *Runtime version* field indicates if an update is available. Select the updated version of Mule runtime engine.
-.. Select *Deploy* to redeploy the application with the updated Mule runtime engine version.
-.. Repeat this for each application running on this Runtime Fabric.
+To upgrade the Runtime Fabric appliance:
 
-. Verify Runtime Fabric is healthy:
-.. Using your web browser, navigate to the *Runtime Fabrics* tab in Runtime Manager and verify that Runtime Fabric reports an *Active* status.
-.. Open a terminal to a Runtime Fabric controller node.
-.. Run `/opt/anypoint/runtimefabric/rtfctl status` and confirm that the cluster status is healthy.
-.. Keep your terminal open to use in later steps.
-
-. Get the latest version of the Runtime Fabric appliance package:
-.. Using your web browser, navigate to Runtime Manager and select the *Runtime Fabrics* tab to view a list of Runtime Fabrics.
-.. Select the *Downloads* link.
-.. Select the *Copy Link* icon next to *Installer package* to copy the code snippet, as shown in the following example:
+. Using your web browser, navigate to Runtime Manager and select the *Runtime Fabrics* tab to view a list of Runtime Fabrics.
+. Select the Runtime Fabric you want to upgrade.
+. Next to the Appliance version, click *Upgrade Available*.
+. Select the *Copy Link* icon next to *Installer package* to copy the code snippet, as shown in the following example:
 +
 ```
 rtfctl appliance upgrade --url https://<upgrade-package-information>
 ```
-.. Paste the code snippet in a text editor for reference. 
+. Paste the code snippet in a text editor for reference. 
+
+=== Update the Mule Deployments Running on Runtime Fabric to the Latest Patch Version
```

Comment:
Did I mention in the other edit that it is too bad that the main body of this topic set is nested at the third heading level? Something to look at another time....